ReactAndroid/src/main/java/com/facebook/react/modules/core/Timing.java

```diff
@@ -155,7 +226,9 @@ private void setChoreographerCallback() {
     if (!mFrameCallbackPosted) {
       Assertions.assertNotNull(mReactChoreographer).postFrameCallback(
           ReactChoreographer.CallbackType.TIMERS_EVENTS,
-          mFrameCallback);
+          mTimerFrameCallback);
+      Assertions.assertNotNull(mChoreographer).postFrameCallback(
```

Comment:
We really don't want to use the regular Choreographer here: ReactChoreographer allows us to be explicit about ordering of callback execution within the React framework, whereas if we use Choreographer directly, we might accidentally execute this before all of the ReactChoreographer callbacks.